client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx

```diff
@@ -185,7 +186,7 @@ const ContactsPrivacyCard = ( props: ContactsCardProps ) => {
 					<ContactDisplay selectedDomainName={ selectedDomainName } />
 					<div className="contact-information__button-container">
 						<Button
-							disabled={ disableEdit || readOnly }
+							disabled={ disableEdit || readOnly || pendingContactUpdate }
 							href={
 								disableEdit || readOnly
```

Comment:
```suggestion
								disableEdit || readOnly || pendingContactUpdate
```
I overlooked this before approving 😬 without this, even if the button has the disabled styling, it still works

Reply:
Good catch!